tools/ovc/openvino/tools/ovc/moc_frontend/pytorch_frontend_utils.py

```diff
@@ -75,7 +75,11 @@ def get_pytorch_decoder(model, example_inputs, args):
     else:
         decoder = model
     args['input_model'] = decoder
-    args["example_input"] = inputs
+    ei = getattr(decoder, "_example_input", None)
```

Comment:
not clear why `_example_input` can happen. Can you please elaborate in the comment?

Reply:
we remove None from example by patching, after that we need to update the example so that type and shape deduction was using it instead of the original one.